src/DynamoPackages/PackageLoader.cs

```diff
@@ -70,40 +72,62 @@ public IEnumerable<IExtension> RequestedExtensions
 
         private readonly List<string> packagesDirectories = new List<string>();
 
-        // Returns the default package directory where new packages will be installed
-        // The default package directory is currently the second entry in the list
-        // The first entry is the standard library
+
+        /// <summary>
+        /// Returns the default package directory where new packages will be installed
+        /// The default package directory is currently the second entry in the list
+        /// The first entry is the standard library.
+        /// </summary>
+        /// <returns>Returns the path to the DefaultPackagesDirectory if found - or null if something has gone wrong.</returns>
         public string DefaultPackagesDirectory
         {
-            get { return packagesDirectories[1]; }
+            get { return packagesDirectories.Count > 1 ? packagesDirectories[1] : null; }
```

Comment:
@sm6srw I took a shot at fixing this like this...it works for my tests - but lets see if it breaks anything else in the tests.

In the long term I don't think this type of solution will work very well - it's leading to bugs like:
https://jira.autodesk.com/browse/DYN-3571
so we may end up changing it soon anyway.

Reply:
Agreed.